src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/userById';
```

Comment:
According to the checklist, data should be prepared in one place, typically the App component, and passed to child components. Avoid using the `import` keyword in non-root components like `NewTodo.tsx`. Consider moving the `usersFromServer` and `getUserById` logic to the App component and passing the necessary data as props to `NewTodo`.